modules/aws/ecs_test.go

```diff
@@ -21,3 +23,33 @@ func TestEcsCluster(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "terratest", *c2.ClusterName)
 }
+
+func TestEcsClusterWithInclude(t *testing.T) {
+	t.Parallel()
+
+	region := GetRandomStableRegion(t, nil, nil)
+
+	client := NewEcsClient(t, region)
+	c1, err := client.CreateCluster(&ecs.CreateClusterInput{
+		ClusterName: aws.String("terratest-extended"),
+		Tags: []*ecs.Tag{&ecs.Tag{
+			Key:   aws.String("test-tag"),
+			Value: aws.String("hello-world"),
+		}},
+	})
+	assert.Nil(t, err)
+
+	defer DeleteEcsCluster(t, region, c1.Cluster)
+
+	assert.Nil(t, err)
+	assert.Equal(t, "terratest-extended", *c1.Cluster.ClusterName)
```

Comment:
Any time you dereference something that comes back from the AWS SDK, please use methods like `aws.StringValue` to avoid possible `nil` pointer dereferences.